Library/Include/Babylon/RuntimeUWP.h

```diff
@@ -1,23 +1,24 @@
 #pragma once
 
-#include "Runtime.h"
-
 #include <windows.ui.core.h>
 #include <windows.ui.xaml.controls.h>
 
-#include <memory>
-
 namespace babylon
 {
+    class Runtime;
+
     class RuntimeUWP final : public Runtime
     {
     public:
         explicit RuntimeUWP(ABI::Windows::UI::Core::ICoreWindow* window, const std::string& rootUrl = {});
-        explicit RuntimeUWP(ABI::Windows::UI::Xaml::Controls::ISwapChainPanel* panel, const std::string& rootUrl = {});
+
```

Comment:
Opening an issue to track the following TODO.

Reply:
Added a line in the Feature Backlog (https://github.com/BabylonJS/BabylonNative/issues/6)